doc/examples/applications/plot_face_detection.py

```diff
@@ -25,7 +25,7 @@
 is specified in the xml file with trained parameters. The ``scale_ratio``
 parameter specifies by which ratio the search window is increased on each
 step. If you increase this parameter, the search time decreases and the
-accuracy decreases. So, faces on some scales can be not detected.
+accuracy decreases. So, faces on some scales cannot be detected.
```

Comment:
Please revert: The meaning is different! The meaning is that, by increasing the step, you take the risk of missing more faces (because there are more scales you don't search).

Reply:
Reverted